🌐 Add German translation for `docs/de/docs/tutorial/security/simple-oauth2.md`

nilslindemann opened the pull request:
← `tutorial/security/get-current-user.md` (#10439)
→ `tutorial/security/oauth2-jwt.md` (#10522)

[German translation progress](https://github.com/tiangolo/fastapi/discussions/10582)

Commit:
🌐 Add German translation for `docs/de/docs/tutorial/security/simple-oauth2.md`

* Full translation
* In sync with current docs/en/docs/tutorial/security/simple-oauth2.md
* Google Translate used
* Manually checked
* Checked for errors using LanguageTool

tiangolo commented:
📝 Docs preview for commit 7e3e8ff0179246f25bb31260a2c19c02df4202b6 at: https://e9f54cee.fastapitiangolo.pages.dev

Commit:
"Das Token" -> "Der Token"

Sources:
https://www.korrekturen.de/genus/der-oder-das-token.shtml
https://rechtschreibung.woxikon.de/artikel/der-oder-das-token

tiangolo commented:
📝 Docs preview for commit 140ea5e373cd4d4febea49687ca884fb863b23b6 at: https://4c784abf.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/tutorial/security/simple-oauth2.md

tiangolo commented:
📝 Docs preview for commit 66502540983c55cbfd4895f8641fb32fb0c3917e at: https://4fab1b37.fastapitiangolo.pages.dev

Commit:
"Nutzer" -> "Benutzer"

tiangolo commented:
📝 Docs preview for commit 244018b38b21863a7bf911f380010a1506b4289f at: https://bced3488.fastapitiangolo.pages.dev

Commit:
Use Typographic quotes

tiangolo commented:
📝 Docs preview for commit f3abc7a434ed42e6058575c0a8e07d9e7e507ba6 at: https://db93bf27.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/tutorial/security/simple-oauth2.md

Commit:
Merge branch 'tiangolo:master' into de/docs/tutorial/security/simple-oauth2.md

tiangolo commented:
📝 Docs preview for commit 4bec42d129fdd5796a5f55521bcd42f2a9c28363 at: https://a607584a.fastapitiangolo.pages.dev

Commit:
Forgot some "Antwort" -> "Response"

tiangolo commented:
📝 Docs preview for commit 797a13309366335f26de672eb4f9053dd37cc68e at: https://ee72ea5a.fastapitiangolo.pages.dev

Commit:
Forgot some "Zeichenfolge" -> "String"

tiangolo commented:
📝 Docs preview for commit fb207e1ce6c3b21c547f82f67f76a3e1e0ad67e0 at: https://8af9751f.fastapitiangolo.pages.dev

Commit:
Wrong pronoun

tiangolo commented:
📝 Docs preview for commit 80f5ccc05c600c8db69526d50b9b02e32120a2fa at: https://4768ba05.fastapitiangolo.pages.dev

Commit:
"das gleiche" vs "dasselbe"

tiangolo commented:
📝 Docs preview for commit 2dc62ab0ecbd9e19f0b09811725706c55aefcfe8 at: https://2ac4d7a9.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/tutorial/security/simple-oauth2.md

Commit:
Merge branch 'tiangolo:master' into de/docs/tutorial/security/simple-oauth2.md

Commit:
"Formular-Body" -> "Formularbody"

tiangolo commented:
📝 Docs preview for commit bfb38e4ade9a2608b88ee7bc2c735fa11718a80d at: https://a697b6d0.fastapitiangolo.pages.dev

Commit:
Merge branch 'tiangolo:master' into de/docs/tutorial/security/simple-oauth2.md

tiangolo commented:
📝 Docs preview for commit 02cbe3294e1fe2b119d40cc967257113bf1bd4b9 at: https://7f03393d.fastapitiangolo.pages.dev

tiangolo commented:
Awesome @nilslindemann! :rocket: 

As you are the champion of German translations and you have done so many, and with so much attention to detail, I'm gonna go ahead and merge this. :tada: 

If others see things that could be improved later, we can check that in new PRs. :nerd_face:
